### PR TITLE
eth staking rewards are now used in accounting

### LINKED ISSUE #5455: ETH staking rewards show as 'missing acquisition' if you try to sell them
## Problem Definition

When creating a profit/loss report for a Rotki account that just has a beacon node validator generating rewards, and a test transaction to sell just those rewards, then the report will have an error that the ETH rewards don't have a corresponding acquisition.

Expected outcome: the report should not have any errors.

I know this is theoretical at this stage because withdrawals are not enabled but I was just testing it out.

### PR DESCRIPTION
**Pull request by prettyirrelevant**
Closes #5455

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`5455` ETH2 staking rewards are now taken into consideration during accounting.
```
Comment (nebolax) [inline docs/changelog.rst:5]: Hmm it is kinda mixed here already, but if I am not mistaken, we tend to keep the list of features on top and the list of bugs in the bottom

File: rotkehlchen/tests/unit/test_history.py [hunk 2]
```diff
@@ -162,3 +163,51 @@ def test_pnl_processing_with_eth2_staking_setting(accountant, db_settings):
...
+    no_message_errors(accountant.msg_aggregator)
+    if db_settings['eth_staking_taxable_after_withdrawal_enabled'] is True:
+        expected_pnls = PnlTotals({
+            AccountingEventType.STAKING: PNL(taxable=ZERO, free=ZERO),
+        })
+    else:
+        expected_pnls = PnlTotals({
+            AccountingEventType.STAKING: PNL(taxable=FVal(473.1710), free=ZERO),
+        })
+    check_pnls_and_csv(accountant, expected_pnls, None)
```
Comment (nebolax) [inline rotkehlchen/tests/unit/test_history.py:213]: Missing new line :(

**Review by nebolax — COMMENTED** [review]
Just a few super minor things

**Review by LefterisJP — COMMENTED** [review]
Hmm I don't think this is the right approach. The test is not bad but it's before we have withdrawals processing.

Think you were right in your initial assesment that nothing is wrong.

I think that the correct approach here is either:
1. Remove ETH2 as an asset completely after a particular upgrade
2. When withdrawals start existing and we add processing for them, then and only then would the accounting make sense. Since at withdrawal ETH2 (so locked ETH) becomes ETH and accounting would match.

File: docs/changelog.rst (hunk 1 above)
Comment (LefterisJP) [reply docs/changelog.rst]: just fyi the order here does not really matter. It's just for us to look nice. It's rendered differently in the actual changelog: https://rotki.readthedocs.io/en/latest/changelog.html

**Comment by LefterisJP** [comment]
Closing this as discussed.

**Pull request closed without merging**